app/javascript/packs/application.js

```diff
@@ -27,3 +27,6 @@ Turbolinks.start()
 const application = Application.start()
 const context = require.context("../src/controllers", true, /\.js$/)
 application.load(definitionsFromContext(context))
+
+// fix for IE Benchmarks doc page Monitoring icon too small/out of alignment #171365469
+$(".benchmark-document .callout-with-icon svg.bar-chart path").attr("transform", "scale(2)");
```

Comment:
this seems pretty specific, is there a more generalized approach to applying transform if needed? this way, the next time we have an svg that needs this, we don't have to remember to do this IE fix.

Reply:
It is specific and that is by intention, it is intended to replace the CSS here:
https://github.com/resolvetosavelives/benchmarks/blob/90cdbc9f1271833ea17b05f7a9108feeba8e05e7/app/assets/stylesheets/benchmark-document.scss#L29

IE needs this because IE does not support SVG transforms via CSS.

And this instance, similar to the CSS, is intended to target only this class of SVG PATH tag.